fleeto_backend/app/controllers/v1/customer/base_controller.rb

```diff
@@ -0,0 +1,26 @@
+class V1::Customer::BaseController < ApplicationController
+  before_action :authenticate_customer!
+  before_action :check_customer_is_verified!
+
+  # quick hack, instead of having to parse the response headers for now when testing the app from postman
+  class NotAuthorized < Exception
```

Comment:
Exception is the root of all the exceptions in Ruby, rather extend from more specific Exception class like RuntimeError or StandardError.
http://rubylearning.com/satishtalim/ruby_exceptions.html